h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -1207,7 +1207,7 @@ private Solver defaultSolver() {
     int max_active = 0;
     if(_parms._family == Family.multinomial )
       for(int c = 0; c < _nclass; ++c)
-        max_active = Math.max(_state.activeDataMultinomial(c).fullN(),max_active);
+        max_active += _state.activeDataMultinomial(c).fullN();
     else max_active = _state.activeData().fullN();
     if(max_active >= 5000) // cutoff has to be somewhere
```

Comment:
Hmm, i still would prefer to make it configurable at least as a hidden system property.

Reply:
That makes sense - at least for experiments.